css/style.css

```diff
@@ -9,358 +9,510 @@
+  background-repeat: no-repeat;
   width: 100%;
+  margin: auto;
+  padding: 20px;
 }
 
-.middle-section .work-content {
-  padding-top: 10px;
-  margin-top: 10px;
-  margin-bottom: 50px;
+section.about .line {
+  width: 100%;
 }
 
-.middle-section .work-content .work-card {
+section.about .card {
   display: flex;
   flex-direction: column;
   justify-content: space-between;
 }
 
-.middle-section .work-content .work-card img {
+section.about .card .card-image img {
   width: 100%;
-  padding: 20px 0 20px 0;
 }
 
-.middle-section .work-content .work-card .card {
+section.about .card .card-detail {
   color: #fff;
 }
 
-.middle-section .work-content .work-card .card h4 {
+section.about .card .card-detail h3 {
   font-family: "Roboto", sans-serif;
   font-size: 32px;
   font-weight: 500;
   padding: 20px 20px 20px 0;
 }
 
-.section-text {
-  font-family: "Inter", sans-serif;
+section.contact .contact-text p {
+  font-size: 16px;
+  line-height: 24px;
   font-weight: 400;
+  font-family: "Inter", sans-serif;
   color: #cacad1;
 }
 
-.middle-section .work-content .work-card .card-text {
+section.about .card .card-detail p {
+  font-family: "Inter", sans-serif;
   font-size: 20px;
+  font-weight: 400;
   line-height: 28px;
-  margin-bottom: 10px;
+  color: #cacad1;
 }
 
-.middle-section .work-content .work-card .cart-link {
+section.about .card .card-detail .card-link {
   padding-top: 20px;
   margin-bottom: 20px;
-  float: right;
 }
 
-.middle-section .work-content .work-card .cart-link ul {
+section.about .card .card-detail .card-link ul {
   display: flex;
   list-style: none;
-  justify-content: space-between;
 }
 
-.middle-section .work-content .work-card .cart-link ul li {
+footer .footer-content p a {
+  color: #fff;
+  font-size: 20px;
+  line-height: 28px;
+  font-weight: 400;
+  font-family: "Roboto", sans-serif;
+}
+
+section.about .card .card-detail .card-link ul li {
   font-family: "Inter", sans-serif;
   font-size: 15px;
   font-weight: 500;
   line-height: 20px;
   border-right: 1px solid #555;
-  padding: 8px 0;
+  padding: 8px 10px;
 }
 
-.middle-section .work-content .work-card .cart-link ul li:last-child {
+section.about .card .card-detail .card-link ul li:last-child {
   border: none;
 }
 
-.middle-section .work-content .work-card .cart-link ul li a {
-  margin-right: 20px;
-}
-
-.btn {
-  background: #7d73fc;
+section.about .card .card-detail .card-link ul li a {
   color: #fff;
-  border: none;
-  width: 100px;
-  height: 50px;
-  margin-bottom: 15px;
-  cursor: pointer;
-}
-
-.btn:hover {
-  background: #7a71f7;
-}
-
-.btn:focus {
-  background: #5e42d7;
-}
-
-.btn:active {
-  background: #7d73fc;
 }
 
-.btn:disabled {
-  background: #2a2b33;
-}
-
-.middle-section .work-content .work-card .btn span {
-  font-family: "Roboto", sans-serif;
-  font-size: 12px;
-  font-weight: 500;
-  letter-spacing: 1px;
-  line-height: 24px;
+section.contact {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
 }
```

Comment:
- [ ] Kindly add a static `width:684px;` to the contact section on the desktop, so that on different screen sizes the contact form's width does not stretch, see video below 👍🏿  



https://user-images.githubusercontent.com/69638013/165936727-46dbe1e0-39a8-4dc6-8563-5641dbd929fa.mp4